ui/src/common/SongDetails.js

```diff
@@ -33,6 +39,13 @@ export const SongDetails = (props) => {
     updatedAt: <DateField record={record} source="updatedAt" showTime />,
     playCount: <TextField record={record} source="playCount" />,
     comment: <MultiLineTextField record={record} source="comment" />,
+    bpm: (
```

Comment:
I think `comment` should always be the last field here, as it can have multiple lines. Maybe put `bpm` after `genre`?